tasks/package.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Set pin content
+  set_fact:
+    pin_content: |
+      {{ pin_content | default('') }}
+      Package: {{ item }}
+      Pin: version {{ redis_version }}
+      Pin-Priority: 1001
+  loop: "{{ redis_packages }}"
+
+- name: Pin Redis release
+  copy:
+    dest: /etc/apt/preferences.d/redis
+    owner: root
+    group: root
+    mode: '0644'
+    content: "{{ pin_content }}"
+  when:
+    - redis_pin_package
+    - ansible_distribution == "Ubuntu" or ansible_distribution == "Debian"
+
+- name: Install Redis package
+  package:
+    name: "{{ redis_packages }}"
+    state: present
+  notify: "disable redis"
```

Comment:
Why disable redis here?

Reply:
When we install redis on debian, it's enabled by default. There are two systemd config, one called `redis-server.service` and another called `redis.service`. I tried to override them with the role but it's not easy. I decided to disable them after installation to be able to install systemd config through the role.

Reply:
It's not a perfect solution but I did'nt find any other ... Feel free to suggest me other one.